dist/animations.js

```diff
@@ -73,7 +73,7 @@ var ANIMATIONS = (_a = {},
             utterancesStartOffset: 0
         },
         progressBar: null,
-        startAnimation: function () { return sampleQuestionAnimation(SAMPLE_QUESTION_ANNA_ANIMATION); }
```

Comment:
Maybe a silly question, I should have asked that before, but why are the JS files version-controlled as well? (as opposed to being generated as part of the build process for instance)

Reply:
No I think that's a fair question!

I wasn't sure how that would all work. When I originally set up the GitHub action to build, I had JS files in the `.gitignore` at first, but then the action would also ignore those files and they wouldn't be added to the repo. We could have the action force them into the commit, but then they would be tracked even with the `.gitignore`.

It's also helpful to have these JS files updated with each commit because it lets us test them in Webflow before merging into `develop` or `main`. Let me know if you have something in mind, it would be great to find a cleaner way to handle this so each PR doesn't have essentially the same files twice, while still allowing us to test in Webflow